lib/LedgerSMB/Report.pm

```diff
@@ -281,8 +281,8 @@ sub _render {
                        $srt_b = $srt_b->to_sort
                            if eval { $srt_b->can('to_sort') };
                    }
-                   no warnings 'numeric';
-                   $srt_a <=> $srt_b or $srt_a cmp $srt_b;
+
+                   $srt_a cmp $srt_b;
```

Comment:
(I think this one needs more discussion, and should probably be pulled out of this PR, with a bug filed instead, if I can give my vote.)